connection.go

```diff
@@ -612,3 +612,14 @@ func (sc *snowflakeConn) SubmitQuerySync(
 
 	return rows.(*snowflakeRows), nil
 }
+
+// TokenGetter is an interface that can be used to get the current tokens and session
+// ID from a Snowflake connection.
+type TokenGetter interface {
+	GetTokens() (token string, masterToken string, sessionID int64)
```

Comment:
Since this info is going out of the driver, could you add a comment explaining what the return values are?

Reply:
Added a comment